### PR TITLE
adding -s option to make test-foreman-rhai in Makefile

### PR DESCRIPTION
**Pull request by anarang**
adding -s option to make test-foreman-rhai in Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -68,13 +68,13 @@ test-foreman-api-threaded:
 	$(PYTEST) $(PYTEST_XDIST_OPTS) $(FOREMAN_API_TESTS_PATH)
 
 test-foreman-cli:
-	$(PYTEST) $(PYTEST_OPTS) $(FOREMAN_CLI_TESTS_PATH)
+	$(PYTEST) $(PYTEST_OPTIONS) $(PYTEST_OPTS) $(FOREMAN_CLI_TESTS_PATH)
```
Comment (oshtaier) [inline Makefile]: `PYTEST_OPTIONS`? Not sure that I see that variable in the list

**Comment by anarang** [comment]
@oshtaier corrected, that was a typo. Just an addition of -s option in the test-foreman-rhai option.

**Comment by oshtaier** [comment]
ACK

**Comment by jyejare** [comment]
ACK

**Pull request merged**